locales/en/fields.json

```diff
@@ -79,6 +79,12 @@
       "court_appearance": {
         "label": "Court"
       },
+      "secure_childrens_home": {
+        "label": "Secure Children's Home (SCH)"
```

Comment:
Quite a minor thing but can we use an apostrophe (`’`) here instead of a prime. To be honest, I was never aware of the difference until somebody pointed it out to me but now it's something I can never un-see! Here's a [good article](https://carsonparkdesign.com/quotation-marks-apostrophes-versus-primes/#:~:text=Single%20primes%20are%20used%20to,and%20double%20primes%20for%20inches.&text=The%20apostrophe%2C%20or%20single%20closed,used%20for%20contractions%20or%20possessives.) explaining the difference.

We've also tried hard to use them correctly everywhere else in app so would be good to maintain that consistency.